doc/users/next_whats_new/annulus.rst

```diff
@@ -0,0 +1,4 @@
+Add ``Annulus`` patch
+---------------------
+
+A new class for drawing elliptical annuli.
```

Comment:
Can this get an example.  A gallery example or adding to an existing one would also be great.  Thanks!

Reply:
Added the example i used in the PR